h2o-docs/src/product/data-science/algo-params/custom_metric_func.rst

```diff
@@ -3,7 +3,7 @@
 ``custom_metric_func``
 ----------------------
 
-- Available in: GBM, DRF 
+- Available in: GBM, DRF, Deeplearning 
```

Comment:
Thank you for filling the documentation - I forgot that and will have to update my PRs :)